### DISCUSSION TITLE
How to install metasploit framework???

### DISCUSSION THREAD

**Discussion by tyrelljspencer** [rapid7/metasploit-framework#18689] [Q&A] [answered]
How to install metasploit

**Comment by bcoles** [comment, marked as answer]
https://www.metasploit.com/get-started

**Comment by Flash7100** [comment]
None of these so called IT gurus can upload a proper working Metasploit framework that actually works. Even the shit from so called Rapid7 is bullshit none is working properly. Ive tried all Metasploit frameworks on Github and Gitlab but none whatsoever works,none! So why fucking bother asking these so called amateurs IT so called gurus even Rapid7 is fucking useless

**Comment by Flash7100** [comment]
None of these so called IT gurus can upload a proper working Metasploit framework that actually works. Even the shiiiit from so called Rapid7 is bullshiiiiit none is working properly. Ive tried all Metasploit frameworks on Github and Gitlab but none whatsoever works,none! So why fucccing bother asking these so called amateurs IT so called gurus even Rapid7 is fucccing useless
